srcs/juloo.keyboard2/KeyboardData.java

```diff
@@ -438,6 +439,15 @@ public static Key parse(XmlPullParser parser) throws Exception
       keysflags |= parse_key_attr(parser, "key6", ks, 6);
       keysflags |= parse_key_attr(parser, "key7", ks, 7);
       keysflags |= parse_key_attr(parser, "key8", ks, 8);
+      /* Compass-point synonyms for swipe gestures */
+      keysflags |= parse_key_attr(parser, "nw", ks, 1);
```

Comment:
I like that the compass-points override the key* attributes but there's a bug: if both `key1` and `nw` are set, the key specified by `nw` is used but the flags of both keys are OR-ed.

Here's a suggestion to fix this bug:
Make a function that checks for one attribute then the other and return that attribute's value. Change `parse_key_attr` to receive a key name instead of an attribute name. Here's how it'll be called:

```java
      keysflags |= parse_key_attr(parser, get_key_attr("nw", "key1"), ks, 1);
```